src/components/OSCALControlImplementationImplReq.js

```diff
@@ -111,9 +111,15 @@ export default function OSCALControlImplementationImplReq(props) {
   if (!implReqStatements) {
     implReqStatements = [];
   }
+
+  // Error check modifications
+  const modAlters = !props.modifications ? null : props.modifications.alters;
+  const modParams = !props.modifications
+    ? null
+    : props.modifications["set-parameters"];
```

Comment:
Is this the same as

```suggestion
  const modAlters = props.modifications?.alters || null;
  const modParams = props.modifications?.["set-parameters"] || null;
```

Which will return `props.modifications.X` _if_ that chain of things exists and if the result is `truthy`. We could drop the `|| null` if this value being an empty array (or undefined) is acceptable.

Reply:
Can we test the falsey path of this? Where these things don't exist? I don't see a test added for that.